person.rb

```diff
@@ -0,0 +1,18 @@
+class Person
+  attr_accessor :id, :name, :age
```

Comment:
- [ Required ] I think it would be better if the `id` attribute in this class does not have a `setter` method but only has a `reader` method as stated in the project requirements. According to the project requirements only the `name` and `age` instance fields should have the `setter` and `reader` methods, I would like to suggest that you refactor this line of your code as shown below.

## Example code
```ruby
  attr_accessor :name, :age
  attr_reader :id
```

## Project requirements 
![Screenshot from 2023-07-24 23-32-15](https://github.com/WilhelmK109/OOP-school-library/assets/16392046/e755809d-acb9-48fb-9a2d-0da75e07fd1b)